tests/OmniSharp.Roslyn.CSharp.Tests/CodeActionsV2Facts.cs

```diff
@@ -256,6 +256,8 @@ public async Task Can_send_rename_and_fileOpen_responses_when_codeAction_renames
                 Assert.Equal(2, changes.Length);
                 Assert.Equal(FileModificationType.Renamed, changes[0].ModificationType);
                 Assert.Contains("Class1.cs", ((RenamedFileResponse)changes[0]).NewFileName);
+                Assert.False(File.Exists(((RenamedFileResponse)changes[0]).FileName), "The old renamed file exists - even though it should not.");
```

Comment:
this is an assert that was missing and helps in refactoring. In the initial run I had all the tests passing, but the rename code action was actually still broken when running in the editor because of the problems with file lifecycles. This is the assert that would have failed